data/sql/updates/pending_db_world/rev_1680443009841196300.sql

```diff
@@ -1,4 +1,6 @@
 -- Ferra loot update
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 14308) AND (`Item` IN (4500, 11414, 11415, 24016));
+
 INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
 (14308, 4500, 0, 1, 0, 1, 0, 1, 1, 'Ferra - Traveler\'s Backpack');
```

Comment:
Btw, where did you get this from?

Reply:
the specific items are from wowhead, which I know we don't use for percentages, so the actual chance values were intended as estimates/placeholders

Reply:
tbf then it won't technically close the original issue. so this still needs some work :p